src/lib/intents.js

```diff
@@ -0,0 +1,139 @@
+import { cozyFetchJSON } from './stack'
+
+/*********************************************
+This following code should never be changed here since it's must be the exact same than in cozy-client-js/src/intents.js
+Service creating functions have been removed since it's not used by the bar
+**********************************************/
```

Comment:
Why not importing / use the cozy-client-js/src/intents.js here rather than duplicate them?